src/esm_runscripts/batch_system.py

```diff
@@ -829,6 +846,12 @@ def calc_launcher_flags(config, model, cluster):
             nproc = config[model]["nproca"] * config[model]["nprocb"]
             cpus_per_proc = 1
             omp_num_threads = 1
+        else:
+
+# kh 22.06.22 defensive (user_error/user_note could also be added here)
+            nproc = 1
+            cpus_per_proc = 1
+#           omp_num_threads = 1
```

Comment:
```suggestion
            nproc = 0
            cpus_per_proc = 0
#           omp_num_threads = 0
```
By making all these 0 if no `command`, the number of `nodes` calculated right below will be 0. This means that for `hdmodel` the number of nodes to be requested is 0 and therefore we won't need the `hdmodel` exception in one of the other changes below in the PR.